kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -366,16 +364,15 @@ def perform_kubelet_query(self, url, verbose=True, timeout=10, stream=False):
         """
         Perform and return a GET request against kubelet. Support auth and TLS validation.
         """
-        with disable_warnings_ctx(InsecureRequestWarning, disable=not self.kubelet_credentials.verify()):
```

Comment:
No need to raise warning log for kubelet?

Reply:
what do you mean ?

Reply:
no warnings